### DISCUSSION TITLE
ToT Admin

### DISCUSSION THREAD

**Discussion by matinnuhamunada** [matinnuhamunada/vm_user_management#3] [Q&A]
## Weekly Goals for Trainer
- *Week 1 Goals*
Pada minggu pertama, kita akan belajar mengenai (1) bagaimana melakukan setting komputer pribadi untuk digunakan dalam proyek bioinformatika, dan (2) mengenal beberapa tools dasar yang akan selalu digunakan dalam analisis: Linux, BASH, Code Editor, dan GitHub.
  - [x] Bikin akun github
  - [x] Install VS Code: https://code.visualstudio.com/
  - [x] Install WSL: https://learn.microsoft.com/en-us/windows/wsl/install
  - [x] Install WSL extension utk VS Code: https://code.visualstudio.com/docs/remote/wsl
  - [x] Mengenal Linux https://youtu.be/rrB13utjYV4
  - [x] Mengenal BASH https://youtu.be/I4EWvMFj37g
  - [x] Mengenal VS Code https://youtu.be/KMxo3T_MTvY

- *Week 2 Goals*
Pada minggu kedua, kita akan praktek (1) menggunakan package manager dan notebook sebagai bentuk dokumentasi proyek bioinformatika, dan (2) mengakses panduan software2 bioinformatika yang ada di GitHub
  - [x] SSH set up: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/about-ssh
  - [x] Conda: https://docs.conda.io/en/latest/miniconda.html
  - [x] Mamba: https://github.com/mamba-org/mamba
  - [ ] Jupyter Lab: https://jupyter.org/
        - [ ] Folder
        - [ ] Environment file
        - [ ] Install environment pake conda
        - [ ] Run jupyter lab
  - [ ] Bioinformatics Project Best Practices

- *Week 3 Goals*
Pada minggu ketiga, kita akan praktek menggunakan fasilitas Cloud Computing
  - [x] Cloud Computing Basic
  - [ ] Creating your own VM
  - [ ] Setting up Linux VM from Scratch
  - [ ] *Hands on project 1*

- *Week 4 Goals*
Pada minggu ke empat, kita akan mencoba menggunakan workflow manager untuk melakukan analisis skala besar
  - [ ] Running a bioinformatics pipeline using workflow manager

### Hands on Project
- [ ] [Bacterial genome assembly and annotation](https://matinnuhamunada.github.io/27255_week_11/02-filtering_and_assembly/index.html)
  - [ ] Mengenal raw data: Fast5
  - [ ] Basecalling: https://github.com/timkahlke/LongRead_tutorials
  - [ ] QC & Filtering
  - [ ] De Novo Assembly
  - [ ] Annotation
  - [ ] Melakukan filtering

**Comment by syahjaya** [comment]
> * _Week 1 Goals_
>   
>   * [x]  Bikin akun github
>   * [x]  Install VS Code: https://code.visualstudio.com/
>   * [x]  Install WSL: https://learn.microsoft.com/en-us/windows/wsl/install
>   * [x]  Install WSL extension utk VS Code: https://code.visualstudio.com/docs/remote/wsl
>   * [x]  Mengenal Linux https://youtu.be/rrB13utjYV4
>   * [x]  Mengenal BASH https://youtu.be/I4EWvMFj37g
>   * [x]  Mengenal VS Code https://youtu.be/KMxo3T_MTvY

harus dichecked kah? saya mau checked list tapi gak bisa :(. jadi pakai ini aja dulu. account di github: done

**Reply by syahjaya** [reply]
eh ternyata bisa :). harus diquote dulu. maaf ya...new be banget
